src/dkr/core/didding.py

```diff
@@ -16,12 +16,12 @@
 from keri.core import coring
 
 DID_KERI_RE = re.compile('\\Adid:keri:(?P<aid>[^:]+)\\Z', re.IGNORECASE)
-DID_WEBS_RE = re.compile('\\Adid:webs:(?P<domain>[^:]+):((?P<path>.+):)?(?P<aid>[^:]+)\\Z', re.IGNORECASE)
+DID_WEBS_RE = re.compile('\\Adid:webs:(?P<domain>[^:]+)(:(?P<port>\d+))?(:(?P<path>.+))?(:(?P<aid>[^:]+)+)\\Z', re.IGNORECASE)
```

Comment:
Hmm according to the [spec](https://trustoverip.github.io/tswg-did-method-webs-specification/#method-specific-identifier):

> A port MAY be included and the colon MUST be percent encoded to prevent a conflict with paths.

Maybe the regex should use `%3A` instead of `:` for parsing the port and avoiding a conflict with the path?